modules/nf-core/glimpse/sample/tests/main.nf.test

```diff
@@ -0,0 +1,62 @@
+nextflow_process {
+
+    name "Test Process GLIMPSE_SAMPLE"
+    script "modules/nf-core/glimpse/sample/main.nf"
+    process "GLIMPSE_SAMPLE"
+    tag "glimpse"
+    tag "glimpse_sample"
```

Comment:
```suggestion
    tag "glimpse/sample"
```

Minor nitpick, otherwise looks good

Reply:
If I understood well this tag need to be the same as the one used in tags.yml
And in the other modules (and the one already merged) we used the name of the tool.

Reply:
I discussed with @sateeshperi  and I will update all the PR to `tool/subtool`